content/applications/sales/sales/invoicing/down_payment.rst

```diff
@@ -1,99 +1,190 @@
 
-- Regular invoice
-- Down payment (percentage)
-- Down payment (fixed amount)
+.. note::
+   Invoices are automatically created in drafts, so they can be reviewed before validation.
+
+On the :guilabel:`Create invoices` pop-up, there are 3 options to choose from in the
+:guilabel:`Create Invoice` field:
+
+- :guilabel:`Regular invoice`
+- :guilabel:`Down payment (percentage)`
+- :guilabel:`Down payment (fixed amount)`
+
+.. note::
+   If :guilabel:`Regular Invoice` is selected, then the other fields disappear, as they only
+   pertain to down payment configurations.
 
 In terms of down payment, it can either be a fixed amount or a percentage of the total amount. The
-first time you request a down payment, you can set a percentage or a fixed amount of your choice,
-and select the right income account and taxes. These settings will be reused for future down
-payments.
+first time a down payment is requested, it can be set to a specific percentage of the total amount,
+or a specific fixed amount.
+
+Once the desired down payment option is selected in the :guilabel:`Create Invoice` field, designate
+the desired amount (either percentage or fixed amount) in the :guilabel:`Down Payment Amount` field.
+
+Then, select the appropriate income account for the invoice in the :guilabel:`Income Account` field,
+and add a tax amount, if necessary, in the :guilabel:`Customer Taxes` field. These settings will be
+reused for future down payments.
 
-.. image:: down_payment/down_payment_1.png
+.. image:: down_payment/create-invoices-popup-form.png
    :align: center
-   :alt: How can you do a down payment on Odoo Sales?
+   :alt: How to configure a down payment on Odoo Sales.
 
 .. important::
-   When you request your first down payment, a new product called **Down payment** will be created.
-   This product will be registered as a **service** product with an invoicing policy of
-   **ordered quantities**. As a reminder, you can edit this product and modify it at any time.
-   Please note that if you choose **delivered quantities** as invoicing policy,
-   **you will not be able to create an invoice**.
-
-Basic sales flow using down payments
-====================================
-
-For this first example, we will use a 50% amount down payment with a product using
-**ordered quantities** as invoicing policy. Make sure to check out our documentation about invoicing
-policies here: :doc:`invoicing_policy` , before requesting your first down payment. When it comes to
-create and view the invoice, you will only have access to a draft invoice mentioning the down
-payment (as you can see below).
-
-.. image:: down_payment/down_payment_2.png
+   When a down payment is first requested, a new product called :guilabel:`Down payment` is created.
+   This product is registered as a :guilabel:`Service` product, with an invoicing policy of
+   :guilabel:`Ordered quantities`. This product can be edited and modified at any time.
+
+.. note::
```

Comment:
I can't tell if this **note** would be better as:
a) a warning block instead, or
b) as a last sentence in the **important** block above it.

Maybe some more context would help, i.e., If delivered quantities is chosen as the invoicing policy— for the original product or the Down payment product?